src/main/resources/assets/not_enough_gamerules/lang/en_us.json

```diff
@@ -17,6 +17,8 @@
   "gamerule.explosionDamage.description": "Set to -1 to use vanilla values",
   "gamerule.keepEffects": "Keep effects",
   "gamerule.keepEffects.description": "Keeps player's effects after death",
+  "gamerule.keepXp": "Keep xp",
+  "gamerule.keepXp.description": "Keeps player's xp after death",
```

Comment:
I think you can override/add default `keepInventory` gamerule description and add the fact that inventory and experience are now separated (something like "Keeps player's inventory after death (no xp)")